package.json

```diff
@@ -62,7 +62,9 @@
     "markdownlint-cli": "0.32.2",
     "prettier": "2.8.0",
     "semver": "7.3.5",
-    "typedoc": "0.22.10",
+    "typedoc": "0.22.18",
+    "typedoc-plugin-missing-exports": "1.0.0",
```

Comment:
Doesn't this add internal details to our docs? Is that wanted? What is it fixing?

Reply:
I think the underlying problem may be that we're not exporting `MetricsAPI` from the API package - that makes the lint step fail as it is not part of the docs. :thinking:

Reply:
Since this was never a problem with the other APIs, there is something being done differently then with metrics that needs to be fixed. I think this plugin is hiding a problem rather than fixing it.

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/3535

Reply:
Some types are not exported but rather their instances are, like in https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-resources/src/detectors/BrowserDetector.ts#L64. It is fine to use those values without their types exported. However, typedoc would explain these and would not document their types and link their base classes.